src/v/kafka/server/requests.cc

```diff
@@ -250,6 +250,8 @@ process_request(request_context&& ctx, ss::smp_service_group g) {
         return do_process<create_acls_handler>(std::move(ctx), g);
     case delete_acls_handler::api::key:
         return do_process<delete_acls_handler>(std::move(ctx), g);
+    case txn_offset_commit_handler::api::key:
+        return do_process<txn_offset_commit_handler>(std::move(ctx), g);
```

Comment:
the request type also has to be hooked up into the api_versions request `kafka/server/handlers/api_versions.cc`

Reply:
that's right but then it will be advertized to clients, I'll add it there when we support all tx api (next PR)